ui/app/styles/helper-classes/layout.scss

```diff
@@ -34,6 +34,10 @@
   position: relative;
 }
 
+.top-xxs {
+  top: $spacing-xxs;
+}
+
```

Comment:
Are we moving away from the `is` pattern, as in `is-top-xxs`?

Reply:
In the rfc it was voted to remove the verb. in this case because there were no related helper classes I didn't use it, but if I'm adding padding, etc. where the used pattern is has-xx, I keep the verb. Not ideal. Basically a state of limbo between what we decided we wanted as a team and what we had time to implement.

Reply:
Sounds good I just couldn't recall. I think we'll be in this in between state until (or if) we do a full overhaul of the classes in this post Bulma world 😁